Tests expect build.config contains 'os: linux'

Two unit tests (test_builds and test_branches) across the build matrix, as they are expecting 'os: linux' in the build.config
 
https://travis-ci.org/menegazzo/travispy/builds/81966563

```
>       assert build.config == {
            'sudo': False,
            '.result': 'configured',
            'os': 'linux',
            'language': 'python',
            'python': [python_version],
            'script': ['py.test'],
        }
E       assert {'.result': '...y.test'], ...} == {'.result': 'c... ['2.6'], ...}
E         Omitting 5 identical items, use -v to show
E         Right contains more items:
E         {'os': 'linux'}
E         Use -v to get the full diff
```

Two months ago the same tests passed using the same codebase, so this must be an API change by Travis-CI.

Do you want to raise this as a regression upstream, or auto-guess the `os` value to maintain the travispy external API?

--- a/travispy/_tests/test_travispy.py
+++ b/travispy/_tests/test_travispy.py
@@ -70,7 +70,6 @@ class Test:
         assert branch.config == {
             'sudo': False,
             '.result': 'configured',
-            'os': 'linux',
             'language': 'python',
             'python': [python_version],
             'script': ['py.test'],
@@ -126,7 +125,6 @@ class Test:
         assert build.config == {
             'sudo': False,
             '.result': 'configured',
-            'os': 'linux',
             'language': 'python',
             'python': [python_version],
             'script': ['py.test'],

